weavedns/README.md

```diff
@@ -153,11 +153,20 @@ link-local as per [RFC6762](https://tools.ietf.org/html/rfc6762),
 
 If DNS is started after you've attached a container to the weave
 network, or you want to give the container a name in DNS *other* than
-its hostname, you can register it using the HTTP API:
+its hostname, you can register it using the `dns-add` command. If no
+FQDN is specified, it is derived from the container's configured
+hostname and domain. For example:
 
 ```bash
 $ docker start $shell2
-$ curl -X PUT "http://$dns_ip:6785/name/$shell2/10.2.1.27" -d fqdn=shell2.weave.local
+$ weave dns-add 10.2.1.27/24 $shell2 -h shell2.weave.local
+```
+
+The inverse operation can be carried out using the `dns-remove` command:
+
+```bash
+$ docker stop $shell2
+$ weave dns-remove 10.2.1.27/24 $shell2
```

Comment:
Hmm. The rationale for doing that isn't clear. weavedns will see the container stop and remove the dns entries anyway. Well, that's what it is supposed to do.

So I think you need a better example here.